docs/platform/10_known-issues.md

```diff
@@ -20,8 +20,7 @@ sidebar_label: Known Issues
 * Downloading artifacts and/or viewing logs from a teammate's experiments is not supported from the CLI. Please use the UI as a workaround.
 
 ### [Datastore](https://docs.grid.ai/features/datastores)
-* grid datastore [create](https://docs.grid.ai/features/datastores/create) ./cifar10
-When you are creating the Gridai datastore. The data directory has some files that are symlinks, if symlinks are softlinks the datastore upload will fail. You need to create a hardlink to those files to upload the datastore.
+* When you are creating the Gridai datastore if the data directory has files that are symlinks, the symlinks that are softlinks will cause the datastore upload to fail. You will need to update to hardlinks to upload to the datastore.
```

Comment:
```suggestion
* When creating a Datastore, data directories that have files that are soft symlinks will cause the datastore upload to fail. Update soft symlinks to hard links in order to prevent this failure.
```
@RobertLaurella

Reply:
```suggestion
* When creating a Datastore, data directories that contain soft symlinks files will cause the Datastore upload to fail. To prevent this failure, update soft symlinks to hard links.
```

Reply:
does "soft symlink(s) files" make sense? I'm trying to streamline that one sentence

@essiequoi

Reply:
yes, that looks good. thanks @RobertLaurella !